### PR TITLE
ACP-383: adjusted PBCs docs

### PR DESCRIPTION
**Pull request by oleksander-kiiashko**
Task: https://spryker.atlassian.net/browse/ACP-383

### CODE REVIEW

**Review by mharacewiat — APPROVED** [review]

File: docs/acp/user/app-composition-platform-installation.md [hunk 1]
```diff
@@ -38,12 +38,15 @@ The actions and level of effort required to make your project ACP-ready may vary
 
 To make your project ACP-ready, different update steps are necessary depending on the template version on which your project was started:
 
-- SCCOS product release [202211.0](/docs/scos/user/intro-to-spryker/releases/release-notes/release-notes-202211.0/release-notes-202211.0.html): All the changes required for ACP readiness are already included, but you should still verify them at the project level.
-- Older versions: To get the project ACP-ready, you should complete all steps described in this document.
+- BEFORE [202211.0](/docs/scos/user/intro-to-spryker/releases/release-notes/release-notes-202211.0/release-notes-202211.0.md) it's complicated and needs an assessment by devs/SA.
```
Comment (evgeniy-it) [inline docs/acp/user/app-composition-platform-installation.md:41]: to remove

**Review by evgeniy-it — DISMISSED** [review]

**Review by lenadoc — CHANGES_REQUESTED** [review]

File: docs/pbc/all/payment-service-provider/202311.0/base-shop/third-party-integrations/payone/integration-in-the-back-office/integrate-payone.md [hunk 2]
```diff
@@ -83,6 +86,309 @@ Alternatively, you may add the domain to the allowlist from the `config/Shared/c
...
+            <process>ItemSupply</process>
+            <process>ItemReturn</process>
+            <process>PaymentRefund</process>
+            <process>PaymentReservationCancel</process>
+            <process>ItemClose</process>
+        </subprocesses>
+...
+```
+
+Main process not only combines and operates sub-processes, its main goal to build everything around the only reserved=”true” state named invoiced;
```
[the remark this answers is not in the document]
Comment (pushokwhite) [reply docs/pbc/all/payment-service-provider/202311.0/base-shop/third-party-integrations/payone/integration-in-the-back-office/integrate-payone.md:348]: That definitely makes no sense for the customers, replacing with the same one we have in Stripe documentation.

File: docs/pbc/all/payment-service-provider/202311.0/base-shop/third-party-integrations/payone/integration-in-the-back-office/integrate-payone.md [hunk 3]
```diff
@@ -83,6 +86,309 @@ Alternatively, you may add the domain to the allowlist from the `config/Shared/c
...
+oms.state.reservation-cancellation-pending,Reservation Cancellation Pending,en_US
+oms.state.reservation-cancellation-pending,Reservation Cancellation Pending,de_DE
+```
+Then run the data import for the glossary:
+
+```bash
+console data:import glossary
+```
+
+### Template changes in `CheckoutPage`
```
[the remark this answers is not in the document]
Comment (pushokwhite) [reply docs/pbc/all/payment-service-provider/202311.0/base-shop/third-party-integrations/payone/integration-in-the-back-office/integrate-payone.md:382]: True, adding `optional` prefix.

**Review by stereomon — DISMISSED** [review]
LGTM

**Review by evgeniy-it — APPROVED** [review]
LGTM

**Pull request merged**